player/api.go

```diff
@@ -1 +1,12 @@
 package player
+
+import (
+	"context"
+
+	"github.com/luno/reflex"
+)
+
+type Client interface {
+	Stream(ctx context.Context, after string, opts ...reflex.StreamOption) (reflex.StreamClient, error)
+	GetRoundParts(ctx context.Context, roundID int64) (RoundInfo, error)
```

Comment:
I think this should include the playerId of the player making the request

Reply:
ok sure